plugins/BEdita/API/src/Controller/Component/JsonApiComponent.php

```diff
@@ -104,6 +104,7 @@ public function getLinks()
     {
         $links = [
             'self' => Router::reverse($this->request, true),
+            'home' => Router::fullBaseUrl() . '/home',
```

Comment:
Here I think we should totally take advantage of named routes: `Router::url(['_name' => 'api:home'], true)`.